aw_client/config.py

```diff
@@ -0,0 +1,16 @@
+from configparser import ConfigParser
+
+from aw_core import dirs
+from aw_core.config import load_config
+
+default_client_config = ConfigParser()
+default_client_config["server"] = {
+    "hostname": "localhost",
+    "port": 5600,
+}
+default_client_config["server-testing"] = {
+    "hostname": "localhost",
+    "port": 5666
+}
+
+client_config = load_config("aw-client", default_client_config)
```

Comment:
I don't like exposing global singletons like this. Perhaps better to just write a function here that wraps `aw_core.load_config`, something like this:

```python
def load_config():
    default_client_config = ConfigParser()
    default_client_config["server"] = {
         "hostname": "localhost",
         "port": 5600,
     }
     default_client_config["server-testing"] = {
         "hostname": "localhost",
         "port": 5666
    }

    return aw_core.config.load_config("aw-client", default_client_config)
```